core/analyze/analyzer.go

```diff
@@ -74,9 +74,12 @@ func variableDetailFromInterface(line string) VariableDetail {
 	nameIndex := regVariable.SubexpIndex("name")
 	typeIndex := regVariable.SubexpIndex("type")
 
+	value := sanitizeVariableValue(groups[typeIndex])
+	values := strings.Split(strings.ReplaceAll(value, " ", ""), "|")
 	return VariableDetail{
-		Name:  sanitizeVariableName(groups[nameIndex]),
-		Value: sanitizeVariableValue(groups[typeIndex]),
+		Name:   sanitizeVariableName(groups[nameIndex]),
+		Value:  value,
```

Comment:
Sollen wir das `Value` komplett ersetzten? (Wenn ich so darüber nachdenke wäre `Types` sprechender)

Weil so gibt es zwei Properties die den (fast) gleichen Inhalt haben 🤔 man könnte ja im Template das joinen mit dem `|` machen 🤔

Reply:
Beide Punkte hab ich mir beim Implementieren auch schon gedacht, war aber dann unschlüssig bezüglich der Webintegration.
Kannst du mich da in die richtige Richtung weisen, wie ich die Webanwendung lokal builden und testen kann?

Reply:
Guter Punkt, ich werde das im readme noch ergänzen.

Schritte:
* im `/core` Folder ist ein Makefile mit einem Target um den core zu builden, also einfach `make build` aufrufen oder den Befehl manuell ausführen, welcher im Target angeführt ist. (dieser Befehl baut den core und legt es gleich richtig im `/docs` folder ab) 
* den `/docs` folder serven, z.b. via `python3 -m http.server 8000` oder direkt über die IDE wenn du z.B.Goland verwendest :)

Der Folder für die UI heißt nrur `docs` damit er als GitHub Page einfach gebaut werden kann

Reply:
Hab `Values` zu `Types` umbenannt und das `Join()` ins Template verlegt.
`Value` hab ich nicht entfernt, da es für Enumwerte verwendet wird.
z.B.
```ts
export enum EnumWithValuesEntity {
  Draft = "draft",
  Published = "published",
}
```
`draft` und `published` wären hier jeweils ein `Value`.

Da `type` ein Keyword in Go ist, hab ich versucht möglichst durchgängig `ty` als Abkürzung zu verwenden, eventuell wär `typ` besser (ähnlich zu `class<->klass` in anderen Sprachen)?

Und zuletzt hab ich noch den Buildoutput von `/web` nach `/docs` verlegt. Oder war das bewusst so gesetzt?

Reply:
Danke ja, das `/web` war noch ein Überbleibsel.

`type` ist ein Keyword ja, aber sowas wie `typeIndex` sollte ja trotzdem gehen oder?

Reply:
Können wir auch machen. Meine Idee war, durchgängig eine Variante zu verwenden damit der Zusammenhang klarer wird.